src/main/java/io/servertap/api/v1/ServerApi.java

```diff
@@ -192,6 +193,83 @@ public static void broadcastPost(Context ctx) {
         ctx.json("true");
     }
 
+    @OpenApi(
+            path = "/v1/scoreboard",
+            summary = "Get information about the scoreboard objectives",
+            tags = {"Server"},
+            responses = {
+                    @OpenApiResponse(status = "200", content = @OpenApiContent(from = Scoreboard.class))
+            }
+    )
+
+    public static void scoreboardGet(Context ctx) {
+        ScoreboardManager manager = Bukkit.getScoreboardManager();
```

Comment:
Is there any chance that this might return `null`? Can a Bukkit-compatible server _not_ have a scoreboard manager?

Reply:
Just checked the docs, they say that there's no scoreboard manager if no worlds are loaded, so pretty much only when the server is still starting up I would think. I'm not sure what kind of response the server should give.